tracer/src/Datadog.Tracer.Native/calltarget_tokens.cpp

```diff
@@ -331,7 +330,7 @@ HRESULT CallTargetTokens::ModifyLocalSig(ILRewriter* reWriter, TypeSignature* me
                                          ULONG* callTargetStateIndex, ULONG* exceptionIndex,
                                          ULONG* callTargetReturnIndex, ULONG* returnValueIndex,
                                          mdToken* callTargetStateToken, mdToken* exceptionToken,
-                                         mdToken* callTargetReturnToken, bool isAsyncMethod)
+                                         mdToken* callTargetReturnToken, std::vector<ULONG>& indexes, bool isAsyncMethod)
```

Comment:
What do you think about renaming `indexes` -> `additionalLocalIndices`?

Reply:
changed in e905bf5a801e21b9552cdd8316deb9e00a6b50c7